metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -97,6 +101,61 @@ class GEProfilerRequest:
     batch_kwargs: dict
 
 
+def get_column_unique_count_patch(self, column):
+    if self.engine.dialect.name.lower() == "redshift":
+        element_values = self.engine.execute(
+            f"SELECT APPROXIMATE count (distinct {column}) FROM {self._table}"
```

Comment:
I'd like to lean on SQLAlchemy where possible. Would something along these lines work?

```suggestion
            sa.select([sa.column(f"APPROXIMATE count (distinct {column})")]).select_from(self._table)
```

Additionally, we may need to modify the `_is_single_row_query_method` in accordance with these changes.

Reply:
I'm not sure if _is_single_row_query_method needs to be changed as all the methods returns similar values than before only some inner method got monkey patched.